HCPCS/manual work/readme.md

```diff
@@ -1,7 +1,23 @@
 ### STEP 8 of the refresh:
 
-8.1. Upload concept_relationship_manual into the working schema (skip this step if implementing on the Pallas vocabulary server).
-Extract the [respective csv file] (https://drive.google.com/drive/u/0/folders/10Gg84mN7tc8d2ByQ9XHNe23rY-w9XBMk) into the concept_relationship_manual table.
+8.1. Upload concept_manual table into the working schema (skip this step if implementing on the Pallas vocabulary server).
+Extract the [respective csv file](https://drive.google.com/drive/u/0/folders/1-Sv1csKD7Bztt62bISdsOhOWZBvJp3SX) into the concept_manual table.
```

Comment:
What is it? We don't share the shredsheets. Only csv of the DDLed tabled to be shared